### DISCUSSION TITLE
Directus 9 - pino-colada outputs gibberish on Windows 10

### DISCUSSION THREAD

**Discussion by ringoldsdev** [directus/directus#3217] [Archive]
```
13:58:56 Γ£¿ Server started at port 8055
```

Consider allowing users to use a different pino prettifier, such as pino-pretty, similar to how you allow users to select database connectors.

Maybe something like
`
LOG_PINO_PRETTIFIER=pino-pretty
`

and have users install it on their own.

**Comment by benhaynes** [comment]
I believe those characters are the emoji, not sure why they don't render on Windows.

**Comment by awesome-developer-20** [comment]
Any updates on this?
Not that it's important but I'm really bored seeing these characters again and again 😅

**Reply by azrikahar** [reply]
Hi, here are the steps to enable Unicode support on Windows (on Windows 10 at least) in order to view them properly:

1. Go to Language settings
2. Click on Administrative language settings
3. In the window that appeared, click on Change system locale
4. Tick the "Beta: Use Unicode UTF-8 for worldwide language support" box
5. Restart your machine

Short clip of above steps on Windows 10, Build 19044.1503:

https://user-images.githubusercontent.com/42867097/151910843-2213089e-1f09-466c-829b-2a1f4f547f71.mp4

**Reply by awesome-developer-20** [reply]
It's working perfectly, thanks 👍
